pandas/tests/test_common.py

```diff
@@ -157,3 +158,10 @@ def test_version_tag():
         raise ValueError(
             "No git tags exist, please sync tags between upstream and your repo"
         )
+
+
+def test_serializable():
```

Comment:
can you parameterize the test on these instead or inside

also pls add the issue number here as a comment

Reply:
I understand. I'll do that.

Reply:
@jreback Could you take a look of this?